frontend/javascripts/oxalis/view/right-menu/tree_hierarchy_view.js

```diff
@@ -70,7 +70,7 @@ type State = {
 
 class TreeHierarchyView extends React.PureComponent<Props, State> {
   state = {
-    expandedGroupIds: {},
+    expandedGroupIds: { [MISSING_GROUP_ID]: true },
```

Comment:
:+1: 

Is this the bug fix you mentioned in the description?

Reply:
No this is just the part that the root group is expanded by default and all other groups are collapsed when loading the tracing.

The bug fix I mentioned is at https://github.com/scalableminds/webknossos/pull/4897/files#diff-4bc7100bdb9fbedf4c2e7cc1aecf60165313748792c78b20dd887ae5a2611ba7R391
All parent groups of a tree get auto-expanded when a tree is activated and scrolled into view. For that we use the "search" functionality of the react-sortable-tree. However, this expansion of the parent groups was not mirrored in the `expandedGroupIds` before as the the `onVisibilityToggle` hook is not executed by the library. This is what I fixed with the linked code.

Reply:
There are a couple more edge-cases, like empty groups inside groups with no trees. I'll see whether I can figure out a good solution for this.

Reply:
Ah, I see. Thanks for the explanation!